BuildConfigurations/StripeiOS Tests-Debug.xcconfig

```diff
@@ -12,3 +12,8 @@ GCC_PREPROCESSOR_DEFINITIONS = DEBUG=1 $(inherited)
 DEBUG_INFORMATION_FORMAT = dwarf
 SWIFT_ACTIVE_COMPILATION_CONDITIONS[arch=*] = DEBUG $(inherited)
 SWIFT_OPTIMIZATION_LEVEL = -Onone
+
+// Hack for getting snapshot tests working on Catalyst
+//FRAMEWORK_SEARCH_PATHS[sdk=macosx*] =
```

Comment:
remove commented out `//FRAMEWORK_SEARCH_PATHS[sdk=macosx*] =` ?

Reply:
Yep this is still in progress, trying to get the tests working in CI.